### ISSUE TITLE
Project name

### ISSUE THREAD

**Issue by wslu42** [qiskit-community/qiskit-hackathon-taiwan-20#6]
# Abstract
Random sequential absorption (RSA) processes for dimers are a typical toy model find great applications in many critical surface science problems including H2 and O2 catalyst efficiency in fuel cells. In this project by generating a 1D RSA simulation code participants are expecting to showcase how to generate a classical algorithm potentially benefits from quantum advantages.

# Description
Let me get the quotes from the wikipedia real quick here:
_Random sequential adsorption (RSA) refers to a process where particles are randomly introduced in a system, and if they do not overlap any previously adsorbed particle, they adsorb and remain fixed for the rest of the process. RSA can be carried out in computer simulation, in a mathematical analysis, or in experiments. It was first studied by one-dimensional models: the attachment of pendant groups in a polymer chain by Paul Flory, and the car-parking problem by Alfréd Rényi.[1]_ 

Ok to be exact, we want to study a 1D discrete Rényi parking problem, which can be found at the end of this paper [2].

The reasons why I propose this project are:
1. The 1D problem has a analytical solution (so we can do sanity check once the code is ready), but in 2D it requires computational efforts nowadays. As a result if we can tackle it in 1D and gain some boost up through quantum advantages, this will greatly benefit the related research fields in surface sciences such as catalyst design in fuel cells.
2. The process of "determined a random site to be bond with dimer" requires random number generator that could be implemented by applying the idea of superposition.
3. The process of "found a certain site was not occupied by another dimer from previous binding process" should be easy to be implemented as a searching task which participants potentially could introduce Grover search.

[1] Rényi, A. (1958). "On a one-dimensional problem concerning random space filling". Publ. Math. Inst. Hung. Acad. Sci. 3 (109–127): 30–36.
[2] MATTHEW P. CLAY AND NANDOR J. SIMANYI (2014) "RENYI’S PARKING PROBLEM REVISITED " https://arxiv.org/pdf/1406.1781.pdf

# Members

 - @githubhandle 
 - @githubhandle - Slack: `@slackhandle` email: `example@example.com`
 - Qiskit Coach: @githubhandle 

# Deliverable
An backend code simulates the 1D dimer RSA process, which should be including below features:
1. Verification of expected coverage rate of dimer binding process to be 1-e^-2 which was found by Page in 1959.
2. Using the idea of superposition to create quantum random number seed to decide which site to be bond with in each iteration.
^^^These are Minimal Viable Products that participants should be able to realize in a 24hr time frame^^^
3. Applying Grover search to this problem, or at least propose a speculation how to design a searching problem that Grover can be implemented in this process.

# GitHub repo

**Comment by ArnieHung** [comment]
Hi, @wslu42.
This project sounds fun! May I join this team?

**Comment by leo07010** [comment]
Hi here, I am interest in this object.
@wslu42.  @ArnieHung 
May I join with u guys?

**Comment by wslu42** [comment]
> Hi, @wslu42.
> This project sounds fun! May I join this team?

Welcome on board :)

**Comment by leo07010** [comment]
Is this related to quantum walk?

**Comment by Lufter** [comment]
I think so, Grover search is a typical application of quantum walk.

**Comment by leo07010** [comment]
Nice, I think maybe I am suitable in this object. Could I join with u?

**Comment by wslu42** [comment]
Welcome! @leo07010

**Comment by leo07010** [comment]
thx, @wslu42

**Comment by ArnieHung** [comment]
Hi @leo07010, I am happy to have you in the team!
Btw, my slack ID is @Arnie, you can add my contact :)

**Comment by leo07010** [comment]
ok,I had already send a message to u

**Comment by leo07010** [comment]
Does anyone commented on question six?LOL

**Comment by bucket0810** [comment]
Hello, @wslu42.
This idea sounds great! May I join this team?

**Comment by leo07010** [comment]
you may discuss with us .
www

**Comment by yaya870611** [comment]
This idea sounds fun! May I join this team?
I'm the chemical engineering student

**Comment by bucket0810** [comment]
@leo07010 where are you guys? there are so many people lol

**Comment by leo07010** [comment]
We are in the middle of this room.You can come to discuss with us.LOL

**Comment by bucket0810** [comment]
Can you stand up or walk to the ground?

**Comment by starktech23** [comment]
@Lufter ~~are you joining this group? You can assign yourself from the top right if you're interested.~~ Never mind, I think you're joining another issue :)

**Comment by yaya870611** [comment]
Sorry, I want to join another group.

**Comment by starktech23** [comment]
@yaya870611 no problem, please comment under that group and remove your assignment from here when you do. Thanks!

**Comment by tom40109** [comment]
may i join this issue?

**Comment by wslu42** [comment]
Welcome folks! I'm running on US east coast time zone so I'm 12hr behind your local time. When you are ready to start introduce ourselves please let me know either here or over Slack. Good luck and have fun!

**Comment by leo07010** [comment]
our work link https://github.com/leo07010/-6
